go.mod

```diff
@@ -1 +1,3 @@
 module gocv.io/x/gocv
+
+go 1.12
```

Comment:
@deadprogram anyway to ignore the auto inclusion of the go version? Not sure what is adding this to this file, if it is go itself or my IDEA (goland).

Reply:
https://youtrack.jetbrains.com/issue/GO-7121

Reply:
So, you will remove that line manually? Seems like that might be your own option at the moment.